src/ontology/foodon-edit.owl

```diff
@@ -18870,6 +18877,63 @@ AnnotationAssertion(dc:date obo:FOODON_00004130 "2021-12-14T21:47:32Z"^^xsd:date
 AnnotationAssertion(rdfs:label obo:FOODON_00004130 "food mixture"@en)
 SubClassOf(obo:FOODON_00004130 obo:NCIT_C45305)
 
+# Class: obo:FOODON_00004147 (marine protein supplement)
+
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_00004147 "A dietary supplement to receive the full nutritional benefits from eating whole fish while eliminating the cooking, tasting and chewing."@en)
```

Comment:
I think this could be simplified to "A dietary supplement made from protein derived from marine animals".